addons/godot-firebase/firestore/firestore_collection.gd

```diff
@@ -115,6 +99,16 @@ func _get_request_url() -> String:
 
 func _process_request(task : FirestoreTask, document_id : String, url : String, fields := "") -> void:
     task.connect("task_error", self, "_on_error")
+    
+    if not auth:
+        printerr("Unauthenticated request issued...")
+        Firebase.Auth.login_anonymous()
+        var result : Array = yield(Firebase.Auth, "auth_request")
+        if result[0] != 1:
+            Firebase.Firestore._check_auth_error(result[0], result[1])
+            return null
+        printerr("Client authenticated as Anonymous User.")
```

Comment:
Should this be printerr? Seems like just a normal print would be best, but up to you, not vital either way.

Reply:
Wanted to make them noticeable, since print may be easily missed and printerr won't block the execution. Do you think that a print is enough?

Reply:
Ohh okay. It should be fine.